README.md

```diff
@@ -1,258 +1,45 @@
-
-The stages in our word count experiment pipeline are configured in `dvc.yaml`. As described in the previous section, this takes the `data/gutenberg` files as input and produces `data/gutenberg_counts.csv` as the final product. Since `data/gutenberg_counts.csv` should be generated whenever the data or scripts change, it is managed by DVC and ignored by git. You can re-run the pipeline steps by running `dvc repro`.
-```
-$ dvc repro
-Running stage 'count-words':
-> python cdstemplate/corpus_counter_script.py data/gutenberg_counts.csv data/gutenberg/*.txt --case-insensitive
-INFO : 2022-05-23 11:18:42,813 : __main__ : Command line arguments: Namespace(csv='data/gutenberg_counts.csv', documents=['data/gutenberg/austen-emma.txt', 'data/gutenberg/austen-persuasion.txt', 'data/gutenberg/austen-sense.txt', 'data/gutenberg/bible-kjv.txt', 'data/gutenberg/blake-poems.txt', 'data/gutenberg/bryant-stories.txt', 'data/gutenberg/burgess-busterbrown.txt', 'data/gutenberg/carroll-alice.txt', 'data/gutenberg/chesterton-ball.txt', 'data/gutenberg/chesterton-brown.txt', 'data/gutenberg/chesterton-thursday.txt'], case_insensitive=True)
-...
-$ dvc repro
-Stage 'count-words' didn't change, skipping
-Data and pipelines are up to date.
+2. Create and activate your environment, rename the environment if you already have an existing one and would like to preserve it. If you want a fresh start, `conda deactivate` then `conda remove -n inatator`. Then run:
 ```
-
-
-You can see the stages in the DAG by running `dvc dag`, in our case it's just a single step called `count-words`:
+conda create -n inatator python=3.10
+conda activate inatator
 ```
-$ dvc dag
-+-------------+
-| count-words |
-+-------------+
-```
-
-## A Note on Notebooks
-We have also included an example Jupyter notebook
 
-Jupyter notebooks are useful tools for exploratory data analysis, prototyping baseline models and creating visualizations. However, they are _not_ an acceptable way to hand-off code for others to reproduce. Have you ever tried to run someone else's notebook, only to find out a cell was deleted, and you have no idea what it was supposed to do?
+3. At the root of the project `pip install -r src/backend/requirements.txt` and `pip install -r src/backend/requirements-dev.txt`.
 
-[Don't put data science notebooks into production](https://martinfowler.com/articles/productize-data-sci-notebooks.html), they are [hard to test, version, parametrize and keep track of state](https://www.reddit.com/r/datascience/comments/ezh50g/jupyter_notebooks_in_productionno_just_no/).
+## :penguin: Run the Application
+Open two terminals. You will need to run the server first, then application frontend.
+1. ***Run the backend (server):*** To start the server, navigate to project root, run `uvicorn src.backend.app.main:app --reload`, to make sure it is working go to `http://localhost:8000/hello/` in your browser.
```

Comment:
What does it mean to 'it is working'? Someone might see the page and not be able to tell if that is what they are supposed to be seeing. Add description or image of what 'working' looks like.